x/concentrated-liquidity/tick_test.go

```diff
@@ -433,3 +435,119 @@ func (s *KeeperTestSuite) TestCrossTick() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestGetLiquidityDepthFromIterator() {
+	firstTickLiquidityDepth := types.LiquidityDepth{
+		TickIndex:    sdk.NewInt(-3),
+		LiquidityNet: sdk.NewDec(-30),
+	}
+	secondTickLiquidityDepth := types.LiquidityDepth{
+		TickIndex:    sdk.NewInt(1),
+		LiquidityNet: sdk.NewDec(10),
+	}
+	thirdTickLiquidityDepth := types.LiquidityDepth{
+		TickIndex:    sdk.NewInt(2),
+		LiquidityNet: sdk.NewDec(20),
+	}
+	fourthTickLiquidityDepth := types.LiquidityDepth{
+		TickIndex:    sdk.NewInt(4),
+		LiquidityNet: sdk.NewDec(40),
+	}
+	tests := []struct {
+		name                   string
+		invalidPool            bool
+		expectedErr            bool
+		lowerTick              int64
+		upperTick              int64
+		exectedLiquidityDepths []types.LiquidityDepth
+	}{
+		{
+			name:      "Full range",
+			lowerTick: firstTickLiquidityDepth.TickIndex.Int64(),
+			upperTick: fourthTickLiquidityDepth.TickIndex.Int64(),
+			exectedLiquidityDepths: []types.LiquidityDepth{
+				firstTickLiquidityDepth,
+				secondTickLiquidityDepth,
+				thirdTickLiquidityDepth,
+				fourthTickLiquidityDepth,
+			},
+		},
+		{
+			name:      "Half range",
+			lowerTick: thirdTickLiquidityDepth.TickIndex.Int64(),
+			upperTick: fourthTickLiquidityDepth.TickIndex.Int64(),
+			exectedLiquidityDepths: []types.LiquidityDepth{
+				thirdTickLiquidityDepth,
+				fourthTickLiquidityDepth,
+			},
+		},
+		{
+			name:      "single range",
+			lowerTick: thirdTickLiquidityDepth.TickIndex.Int64(),
+			upperTick: thirdTickLiquidityDepth.TickIndex.Int64(),
+			exectedLiquidityDepths: []types.LiquidityDepth{
+				thirdTickLiquidityDepth,
+			},
+		},
+		{
+			name:                   "tick that does not exist",
+			lowerTick:              10,
+			upperTick:              10,
+			exectedLiquidityDepths: []types.LiquidityDepth{},
+		},
+		{
+			name:        "Half range",
```

Comment:
I think this name is wrong and should be something like "non-existent pool"

Reply:
Thanks for the catch! Forgot to push my commit of editing this, resolved in the most recent commit